projects/plugins/jetpack/_inc/client/search/dashboard/index.jsx

```diff
@@ -8,19 +8,30 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+
 import { JetpackFooter } from '@automattic/jetpack-components';
 import restApi from '@automattic/jetpack-api';
-import Masthead from 'components/masthead';
+/* eslint-disable no-duplicate-imports */
+import { JetpackLogo } from '@automattic/jetpack-components';
```

Comment:
I think my one minor comment didn't actually get posted :)

I wanted to ask if we could switch to

`import { JetpackLogo, JetpackFooter } from '@automattic/jetpack-components';`

Reply:
Ah silly me I did try to look for the comment but didn't find any and just assumed the comment above was the comment 😀
I'll change that in a separate PR. Good catch thanks.